cmd/sink/logs/logs.go

```diff
@@ -26,19 +26,19 @@ const (
 func getKubeArchiveLoggingConfig() (map[string]string, error) {
 	loggingDir, exists := os.LookupEnv(files.LoggingDirEnvVar)
 	if !exists {
-		return nil, errors.New("Environment variable not set: " + files.LoggingDirEnvVar)
+		return nil, errors.New("environment variable not set: " + files.LoggingDirEnvVar)
 	}
 	configFiles, err := files.FilesInDir(loggingDir)
 	if err != nil {
-		return nil, fmt.Errorf("Could not read logging config: %w", err)
+		return nil, fmt.Errorf("eould not read logging config: %w", err)
 	}
 	if len(configFiles) == 0 {
-		return nil, errors.New("Logging Config is empty. To configure logging update the kubearchive-logging ConfigMap")
+		return nil, errors.New("logging Config is empty. To configure logging update the kubearchive-logging ConfigMap")
```

Comment:
if logging is lower case, I think you should make Config lower case too

```suggestion
		return nil, errors.New("logging config is empty. To configure logging update the kubearchive-logging ConfigMap")
```

Reply:
I changed the error to the one suggested by Greg